src/Curl/Multi.jl

```diff
@@ -55,7 +55,7 @@ end
 function remove_handle(multi::Multi, easy::Easy)
     lock(multi.lock) do
         @check curl_multi_remove_handle(multi.handle, easy.handle)
-        deleteat!(multi.easies, findlast(==(easy), multi.easies))
+        deleteat!(multi.easies, findlast(==(easy), multi.easies)::Integer)
```

Comment:
```suggestion
        deleteat!(multi.easies, findlast(==(easy), multi.easies)::Int)
```
Might as well give a concrete type here.